.github/workflows/test-sui.yaml

```diff
@@ -1,16 +1,36 @@
 name: Test Sui
 
-on:
-  pull_request:
-    paths:
-      - sui/**
-      - common/**
-      - .github/actions/setup-sui/**
-      - .github/workflows/test-sui.yaml
+on: pull_request
 
 jobs:
+  check-relevant-changes:
```

Comment:
Keep the two as a single job and add this as the first step? Looks a bit cleaner on the checks